espanol/bombas/bombas.md

```diff
@@ -0,0 +1,7 @@
+Grandes bombas caían sobre todo Canudos. Hombres, mujeres, niños y ancianos; todos eran consumidos por las explosiones y el fuego, un fuego tan intenso que no dejaba nada a su paso.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.